tools/jenkins/run_full_performance.sh

```diff
@@ -34,7 +34,7 @@ git clean -fdxq --exclude='report*.xml'
 
 # scalability with 32cores (and upload to a different BQ table)
 tools/run_tests/run_performance_tests.py \
-    -l c++ java csharp go \
+    -l c++ java csharp go php7 php7_protobuf_c \
```

Comment:
Hi, @jtattermusch . Is simply add php7 here I can get the benchmark result inside 32 cores environment?

Reply:
Yes (if the 32core workers have the right prerequisites installed. just saying - ideally all the workers should have identical setup).

From a practical perspective, running PHP benchmarks on 32core machines (as opposed to 8core) makes a little sense IMHO. It's useful for languages designed for scalability and high performance (C++, java, go, C# I think). So I don't think there's any reason to run PHP on 32core setup - it will only make the suite take longer.

Reply:
Since in php, each channel uses one process. I tested in 32 cores by adhoc and it can use almost all cores in it and streaming QPS can reach 900,000+.
Also, It is the ruby_proxy who created those php processes and close them after the test finish. It's really dangerous if it breaks, those php process will remain in the system and consume lots of CPU. Run "pkill -9 php" everytime after the test can avoid this, but I haven't find a suitable place to add this.